kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -156,4 +161,12 @@ public interface Constants {
     String DONT_USE_KRAFT_MODE = "-UseKRaft";
     String USE_KAFKA_NODE_POOLS = "+KafkaNodePools";
     String DONT_USE_KAFKA_NODE_POOLS = "-KafkaNodePools";
+
+    /**
+     * Scraper pod labels
+     */
+    String SCRAPER_LABEL_KEY = "user-test-app";
+    String SCRAPER_LABEL_VALUE = "scraper";
+    String SCRAPER_NAME = "scraper";
+    String DEPLOYMENT_TYPE = "deployment-type";
```

Comment:
```suggestion
    String DEPLOYMENT_TYPE_LABEL_KEY = "deployment-type";
```
and could make the `Scraper` deployment type a constant (or use SCRAPER_NAME)